src/improvements/tasks/MultisigTask.sol

```diff
@@ -411,6 +412,11 @@ abstract contract MultisigTask is Test, Script, StateOverrideManager {
 
             vm.writeFile(vars.filepath, json);
             console.log("\nWrote verify input to %s", vars.filepath);
+            string memory base64Json = Base64.encode(bytes(json));
+            console.log(
+                "\nTo verify this transaction, run `go run ./cmd/op-verify qr` on your machine, then open the following link on your mobile device: https://op-verify.optimism.io/?tx=%s",
```

Comment:
```suggestion
                "\nTo verify this transaction, run `op-verify qr` on your machine, then open the following link on your mobile device: https://op-verify.optimism.io/?tx=%s",
```

Making this edit since users will just have the binary instead of the full repository